cmd/jaeger/internal/exporters/storageexporter/exporter.go

```diff
@@ -50,5 +50,10 @@ func (*storageExporter) close(_ context.Context) error {
 }
 
 func (exp *storageExporter) pushTraces(ctx context.Context, td ptrace.Traces) error {
-	return exp.traceWriter.WriteTraces(ctx, exp.sanitizer(td))
+	// Make sure exp.sanitizer func work when multiple exporters are defined in pipeline
+	// More info see: https://github.com/open-telemetry/opentelemetry-collector/blob/main/internal/fanoutconsumer/traces.go#L66-L75
+	newTraces := ptrace.NewTraces()
+	td.CopyTo(newTraces)
```

Comment:
copying should only happen if necessary, not forced for every user who may not be running dual exporters

Reply:
we could push the copying all the way down to sanitizers